examples/widgets/slider_demo.py

```diff
@@ -5,63 +5,70 @@
 
 Using the slider widget to control visual properties of your plot.
 
-In this example, a slider is used to choose the frequency of a sine
-wave. You can control many continuously-varying properties of your plot in
-this way.
+In this example, sliders are used to control the frequency and amplitude of
+a sine wave. You can control many continuously-varying properties of your plot
+in this way.
+
+For a more detailed example of value snapping see
+:doc:`/gallery/widgets/slider_snap_demo`.
+
+For an example of using a `matplotlib.widgets.RangeSlider` to define a range
+of values see :doc:`/gallery/widgets/range_slider`.
 """
 import numpy as np
 import matplotlib.pyplot as plt
-from matplotlib.widgets import Slider, Button, RadioButtons
+from matplotlib.widgets import Slider, Button
+
+
+def fxn(t, amp, freq):
```

Comment:
What does `fxn` stand for? I'd use one of `f`, `func`, `sine`, `wave`, ...

Also maybe add a comment
~~~
# The parametrized function to be plotted
~~~

Reply:
I use `fxn` as yet another abbreviation for `function` but if that potentially unclear to some readers then it shouldn't be in an example. I'll use `f` or `func`. Also :+1: to the comment.

In keeping with the more explicit names of the sliders I'll also change the arguments to their long form names.